doc/release-notes/0.18/0.18.md

```diff
@@ -0,0 +1,172 @@
+<td valign="top">This feature was disabled by default in Eclipse OpenJ9 0.17.0 due to issue <a href="https://github.com/eclipse/openj9/issues/7332">#7332</a>. This issue is resolved and the option is reenabled.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/7427">#7427</a></td>
+<td valign="top">Support for Terabyte suffixes for <tt>-X</tt> and <tt>-XX</tt> options</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">OpenJ9 now supports 't' and 'T' suffixes (indicating terabytes) for <tt>-X</tt> and <tt>-XX</tt> options that take a <tt>&lt;size&gt;</tt> parameter.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7728">#7728</a></td>
+<td valign="top">Extended platform support for pause-less garbage collection</td>
+<td valign="top">OpenJDK8 and later (macOS)</td>
+<td valign="top">Support for pause-less garbage collection (<tt>-Xgc:concurrentScavenge</tt>) is now extended to macOS systems.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7955">#7955</a></td>
+<td valign="top">Remove restriction on multi-layer shared cache</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">In the previous release, there is a restriction that the <tt>jvmtiSharedCacheInfo.isCorrupt</tt> field and the  <tt>SharedClassCacheInfo.isCacheCorrupt()</tt> method cannot detect a corrupted cache that has a layer number other than zero. This restriction is now removed.</td>
+</tr>
+
+</table>
+
+
+## Known Issues
+
+The v0.17 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1">
+<thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/7459">#7549</a></td>
+<td valign="top">Startup regression on AIX</td>
+<td valign="top">AIX (64-bit POWER)</td>
+<td valign="top">A 40 - 80% startup regression is under investigation for OpenJDK 8 and 11.</td>
+<td valign="top">None</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/479">#479</a></td>
+<td valign="top">Non-compressed references VM builds not available</td>
+<td valign="top">AIX, Linux on Z, Linux on Power</td>
+<td valign="top">Non-compressed references VM builds, which support heaps larger than 57GB, are built on Linux on x86, Windows, and macOS. </td>
+<td valign="top">Manual builds on other platforms are possible by following our <a href="https://github.com/eclipse/openj9/blob/master/buildenv/Build_Instructions_V8.md">detailed build instructions</a>.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/897">#897</a></td>
```

Comment:
These last two items about javadoc should be resolved and can be removed. I'll see about closing the issues.

Reply:
Actually, they are only resolved for Java 11+.

Reply:
Added (OpenJDK 8 only)